openquake/commonlib/calculators/event_based_risk.py

```diff
@@ -61,29 +56,49 @@ def event_loss(riskinputs, riskmodel, rlzs_assoc, monitor):
                         for asset, loss, ins_loss in zip(
                             out.assets, losses, ins_losses)
                         if loss and asset.id in specific]
-                acc[out.hid, out.loss_type] += {
-                    tag: AccumDict(
-                        data=data, loss=sum(losses), ins_loss=sum(ins_losses),
-                        nonzero=sum(1 for loss in losses if loss),
-                        total=len(losses))}
+                ad = AccumDict(
+                    data=data, loss=sum(losses), ins_loss=sum(ins_losses),
+                    nonzero=sum(1 for loss in losses if loss),
+                    total=len(losses))
+                try:
+                    a = acc_rlz[out.loss_type, tag]
+                except KeyError:
+                    acc_rlz[out.loss_type, tag] = ad
+                else:
+                    a += ad
```

Comment:
The above names could be made more explicit (a, ad, acc, acc_rlz, ecc.), or some further comments might be added